omnigibson/omnigibson_4_0_0.kit

```diff
@@ -22,7 +22,7 @@ keywords = ["experience", "app", "usd"]
 "omni.isaac.kit" = {}
 "omni.isaac.lula" = {}
 "omni.isaac.manipulators" = {}
-"omni.isaac.ml_archive" = {}
+## "omni.isaac.ml_archive" = {}
```

Comment:
what's the reason for not using these imports?

Reply:
They are actually excluded in the kit file shipped in pip-installed Isaac Sim, both in the kit file and in the actual exts directory, meaning if we keep these OG won't work with the pip installed version (those extensions cant be found). But these I believe are some off-the-shelf models etc. that we don't use, looks very safe to remove.